doc/ipython-notebooks/converter/HessianLLE.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "#The scikit-example for this notebook\n",
+    "#is here - http://scikit-learn.org/stable/auto_examples/manifold/plot_lle_digits.html\n",
+    "\n",
+    "from scipy.io import loadmat, savemat\n",
+    "from numpy    import random\n",
+    "from os       import path\n",
+    "import matplotlib.pyplot as plt\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "\n",
+    "\n",
+    "mat  = loadmat('usps.mat')\n",
+    "Xall = mat['data']\n",
+    "Xall_ = np.resize(Xall, (256, 1024))\n",
+    "#The number of training examples have been reduced to make it feasible to run.\n",
+    "\n",
+    "#print Xall_\n",
+    "#print Xall_.shape\n",
+    "Yall = np.array(mat['label'].squeeze(), dtype=np.double)\n",
+    "Yall_ = np.resize(Yall, (1024, ))\n",
+    "def plot(data, embedded_data, colors='m'):\n",
```

Comment:
please define functions in their own cells
Also give every function a proper name, and put a minimal docstring